test/dom/unforgeables.spec.js

```diff
@@ -0,0 +1,27 @@
+import createSecureEnvironment from '../../lib/browser-realm.js';
+
+const evalScript = createSecureEnvironment(undefined, window);
+
+describe('EventTarget unforgeable', () => {
+    it('should be accessible from window', function() {
+        // expect.assertions(4);
+        evalScript(`
+            expect(EventTarget !== undefined).toBe(true);
+            expect(window.__proto__.__proto__.__proto__ === EventTarget.prototype).toBe(true);
+            expect(document.body instanceof EventTarget).toBe(true);
+            expect(document.createElement('p') instanceof EventTarget).toBe(true);
+        `);
+    });
+});
+
+describe('Window unforgeable', () => {
+    it('should be accessible from window', function() {
+        // expect.assertions(4);
+        evalScript(`
+            expect(Window !== undefined).toBe(true);
```

Comment:
adding similar tests for `Window` unforgeable, which I think has a very similar problem, maybe we should do the same there... although it is a lot less common, and detaching the iframe has no effects on this global name.